templates/contribute/style.md

```diff
@@ -392,6 +392,21 @@ We generally use a blank line to separate theorems and definitions,
 but this can be omitted, for example, to group together a number of
 short definitions, or to group together a definition and notation.
 
+### Hypotheses Left of Colon ###
+
+Generally, having hypotheses to the left of the colon is preferred
+over having hypotheses in implications, when possible. For instance:
```

Comment:
```suggestion
Generally, having arguments to the left of the colon is preferred
over having arguments in universal quantifiers or implications, 
if the proof starts by introducing these variables. For instance:
```
I think it's good to make this rule only apply if the proof starts by introducing the variable, (e.g. by `λ` or by `intro`). It's fine to keep the argument on the right if the first step of the proof is something else, e.g. induction.